### ISSUE TITLE
icon duplicates

### ISSUE THREAD

**Issue by esrasutton** [phoenixlauncher/phoenix#129]
Editing the icon duplicates its image to other games.

**Comment by jxhug** [comment]
version? can you provide more context such as what menu this was in and give logs please?
logs are found in `~/Library/Application Support/Phoenix`

**Comment by Shock9616** [comment]
Yeah what jx said. Please double check that you're using the latest version (v0.1.4) with`Phoenix > About Phoenix` in the menu bar. If you _are_ on the latest version, we'll need a more detailed description of what you're doing (a screen recording would be even better!) and the logs to be able to help.

**Comment by Shock9616** [comment]
Just noting that today I noticed a couple weird duplicate icons as well (on v0.1.4). Haven't nailed down what I did differently to cause the issue, but it _is_ there on latest version.
